docs/UserGuide.md

```diff
@@ -161,6 +161,16 @@ Format: `exit`
 
 AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
+If you want to save the data to somewhere else or rename it into a new file, you can also save the data by using the `save` command.
+
+Format: `save /dir DIRECTORY /name FILE_NAME`
```

Comment:
Can you change the format to this thanks!
save /dir [directory] /name [name of the JSON file]